test/powershell/Language/Scripting/HashtableToPSCustomObjectConversion.Tests.ps1

```diff
@@ -129,5 +137,21 @@
 	    $obj.PSTypeNames.Count | Should Be 3
 	    $obj.PSTypeNames[0] | Should Be 'System.Object'
     }
+    It "new-object should fail to create object for System.Management.Automation.PSCustomObject" {
+
+        $errorObj = $null
+        $obj = $null
+		$ht = @{one=1;two=2}
+        try
+        {
+            $obj = New-Object System.Management.Automation.PSCustomObject -property $ht
+        }
+        catch
+        {
+            $errorObj = $_
+        }
+        $obj | should be $null
+        $errorObj.FullyQualifiedErrorId | should be "CannotFindAppropriateCtor,Microsoft.PowerShell.Commands.NewObjectCommand"

```

Comment:
Should we add a follow-up item on that to completely remove this type?
cc @lzybkr